projects/1.16/assets/portality/portality/lang/zh_cn.json

```
@@ -1 +1,51 @@
-{}
\ No newline at end of file
+{
+    "itemGroup.portality": "Portality",
+    "block.portality.controller": "传送门控制器",
+    "block.portality.frame": "传送门框架",
+    "block.portality.module_interdimensional": "多维度交互模块",
+    "block.portality.module_items": "物品模块",
+    "block.portality.module_fluids": "流体模块",
+    "block.portality.module_energy": "能量模块",
+    "block.portality.generator": "低效能发电机",
+    "portality.controller.error.size": "错误：目前的传送门结构是无效的！",
+    "portality.controller.error.privacy": "错误：此传送门是私有的！",
+    "portility.controller.info.icon_changed": "传送门图标已更改！",
+    "portality.display.private": "私有",
+    "portality.display.public": "公共",
+    "portality.display.dial": "拨号链接",
+    "portality.display.dial_once": "拨号链接一次",
+    "portality.display.force": "强制链接",
+    "portality.display.call_portal": "调用传送门",
+    "portality.display.close_portal": "关闭传送门",
+    "portality.display.change_name": "更改传送门名称",
+    "portality.display.make_public": "将传送门转为公共",
+    "portality.display.make_private": "将传送门设为私有",
+    "portality.display.hide_name": "隐藏名称显示",
+    "portality.display.show_name": "显示名称显示",
```

```suggestion
    "portality.display.hide_name": "隐藏名称",
    "portality.display.show_name": "显示名称",
```
https://github.com/CFPAOrg/Minecraft-Mod-Language-Package/blob/f96ab8a3b30d898cf24e9b80615772b5d2b55c6c/projects/1.12.2/assets/portality/portality/lang/zh_cn.lang#L13
https://github.com/CFPAOrg/Minecraft-Mod-Language-Package/blob/f96ab8a3b30d898cf24e9b80615772b5d2b55c6c/projects/1.12.2/assets/portality/portality/lang/zh_cn.lang#L18
原文：
  "portality.display.hide_name": "Hide Name Display",
  "portality.display.show_name": "Show Name Display",